src/components/Carousel/Carousel.tsx

```diff
@@ -51,6 +53,17 @@ export const Carousel: FunctionComponent<CarouselProps> = ({ slides }) => {
         const { title, subheader, description, backgroundImage, buttonYoutube, buttonPage, buttonDownload } = slide;
         const hasCta = buttonYoutube || buttonPage || buttonDownload;
         const styleSlide = backgroundImage ? { backgroundImage: `url("${backgroundImage}")` } : {};
+        const downloadDocument = () => {
+          if (!buttonDownload) {
+            return;
+          }
+          gaEvent({
+            category: GaCategory.FILE_DOWNLOAD,
+            action: GaAction.CAROUSEL_DOWNLOAD,
+            label: buttonDownload.file,
+          });
+          window.open(buttonDownload.file, "_blank", "noopener,noreferrer");
+        };
```

Comment:
did u check in the analytics console after event fired?

Reply:
how about keeping the prev href and onClick fire directly GA tracking instead? so u:
- avoid the unnecessary [conditional check](https://github.com/TradeTrust/tradetrust-website/pull/648/files#diff-6bc44e73ed9ce117006101e81c17a4dbaffe15c4d136b00a01aeb4720a9e9e2fR57-R59) and manual open new tab.
- adhere to the principles of unobtrusive js.

if you want to have a new tab approach, use `target="_blank"` directly? let users ownself download themselves in the tab pdf view.